### DISCUSSION TITLE
取消视频进度记忆和自动开启洗脑循环

### DISCUSSION THREAD

**Discussion by archerzhoujj** [the1812/Bilibili-Evolved#4560] [功能建议 / Ideas]
取消视频进度记忆：每次点进视频都从头开始
自动开启洗脑循环：每个视频都循环播放

**Comment by Geeyun-JY3** [comment]
请善用搜索，重复：
- > 取消视频进度记忆：每次点进视频都从头开始

  #1071、#1191

- > 自动开启洗脑循环：每个视频都循环播放

  #987
